Opcodes/fout.c

```diff
@@ -399,22 +396,20 @@ static int32_t outfile_set_S(CSOUND *csound, OUTFILE *p)
   int32_t istring = 1;
   STDOPCOD_GLOBALS *pp = (STDOPCOD_GLOBALS*) csound->QueryGlobalVariable(csound,
                                                                          "STDOPC_GLOBALS");
-  OPARMS oparms;
-  csound->GetOParms(csound, &oparms);
   
   memset(&sfinfo, 0, sizeof(SFLIB_INFO));
   format_ = (int32_t) MYFLT2LRND(*p->iflag);
   if (format_ >= 51)
     sfinfo.format = AE_SHORT | TYP2SF(TYP_RAW);
   else if (format_ < 0) {
-    sfinfo.format = FORMAT2SF(oparms.outformat);
-    sfinfo.format |= TYPE2SF(oparms.filetyp);
+    sfinfo.format = FORMAT2SF((csound->GetOParms(csound))->outformat);
```

Comment:
A lot of repeated csound->GetOparms(csound) here in this file. Seems better to have:

```OPARMS oparms;
  oparms = csound->GetOParms(csound);
```
  
once and use oparms throughout this function. (Or 'O' if we want to match up with usage elsewhere.)

Reply:
Fixed.